src/python/grpcio/grpc/_cython/_cygrpc/aio/call.pyx.pxi

```diff
@@ -125,7 +125,7 @@ cdef class _AioCall(GrpcCallWrapper):
         if credentials is not None:
             set_credentials_error = grpc_call_set_credentials(self.call, credentials.c())
             if set_credentials_error != GRPC_CALL_OK:
-                raise Exception("Credentials couldn't have been set")
+                raise RuntimeError("Credentials couldn't have been set: {0}".format(set_credentials_error))
```

Comment:
nit: Can you use the new `InternalError` instead of `RuntimeError`? If you are not synced with master, feel free to skip this one.